### DISCUSSION TITLE
Possibility to change text font.

### DISCUSSION THREAD

**Discussion by SergioXp** [certinia/debug-log-analyzer#17] [Ideas] [answered]
Hi everyone,

Is it possible to put the option to change the text font? if you have a little font size is very hard to read.

<a href="https://imgbb.com/"><img src="https://i.ibb.co/K2bG7zv/image.png" alt="image" border="0"></a>

Thanks!

**Comment by kjonescertinia** [comment]
We should be picking that font size up from VSCode settings so you can adjust that way but maybe we can also add a way to override as well, perhaps a specific setting might be a useful option for that so your override can be saved.

**Comment by lcottercertinia** [comment, marked as answer]
@SergioXp The font size is inherited from the vscode default (which you can change) but we also found a couple of places we were not scaling the font correctly from this default. This should be fully resolved in the v1.6 release.
#239
